superset/db_engine_specs/base.py

```diff
@@ -1445,7 +1445,19 @@ def validate_parameters(
         port = parameters.get("port", None)
         if not port:
             return errors
-        if not is_port_open(host, port):
+        if not (isinstance(port, int) and 0 <= port < 2 ** 16):
```

Comment:
do we stringify the json? @hughhhh 

![Screen Shot 2021-06-18 at 11 18 45 AM](https://user-images.githubusercontent.com/48933336/122583419-34055a00-d027-11eb-8d97-dc68702480a0.png)

![Screen Shot 2021-06-18 at 11 20 24 AM](https://user-images.githubusercontent.com/48933336/122583430-37004a80-d027-11eb-8707-c49375f80e2f.png)

Reply:
This number should be an int, let me look on the frontend